src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -1,40 +1,113 @@
+                    <label class="font-weight-bold" [innerHTML]="criterion!.title! | htmlForMarkdown"></label>
+                    <table class="table table-bordered">
+                        <thead>
+                            <tr>
+                                <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.credits">Credits</td>
+                                <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.gradingScale">Grading Scale</td>
+                                <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.instructionDescription">Instruction Description</td>
+                                <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.feedback">Feedback</td>
+                                <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.usageCount">Max Count in Score</td>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr *ngFor="let instruction of criterion.structuredGradingInstructions">
+                                <td style="width: 5%" [innerHTML]="instruction.credits"></td>
+                                <td style="width: 10%" [innerHTML]="instruction!.gradingScale! | htmlForMarkdown"></td>
+                                <td style="width: 40%" [innerHTML]="instruction!.instructionDescription! | htmlForMarkdown"></td>
+                                <td style="width: 35%" [innerHTML]="instruction!.feedback! | htmlForMarkdown"></td>
+                                <td style="width: 10%" [innerHTML]="instruction.usageCount"></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </ng-container>
+        <!-- End of Preview-->
+    </jhi-markdown-editor>
+</div>
+
+<div class="edit-sqi" *ngIf="exercise.gradingInstructionFeedbackUsed">
+    <div class="grading-instructions-update-border">
+        <div *ngFor="let criteria of exercise.gradingCriteria">
+            <div class="criteria-content" style="padding-bottom: 12px">
+                <div class="row">
+                    <div class="col-md-10 text-left">
+                        <div class="input-group" style="height: 33px; width: 75%">
+                            <input
+                                class="form-control"
+                                type="text"
+                                [(ngModel)]="criteria.title!"
+                                (change)="onCriterionTitleChange($event, criteria)"
+                                placeholder="Criterion Title"
+                            />
+                            <span class="col-auto input-group-btn d-flex align-items-center" id="reset-criteria-button" (click)="resetCriterionTitle(criteria)">
+                                <fa-icon [icon]="'undo'"></fa-icon>
```

Comment:
IMO, You should add a `cursor: pointer;` css declaration for this icon.

Reply:
oh, cool.. that's good to know 👍